### ISSUE TITLE
Naming of "pad" feature field

### ISSUE THREAD

**Issue by YoshuaNava** [ethz-asl/libpointmatcher#415]
While working with libpointmatcher I have found the "pad" feature field to be difficult to understand by the name.

Knowing that libpointmatcher represents all points in homogeneous coordinates, and considering that "pad" seems to refer to the scale component, I would propose that we make its meaning clearer, by either:

1) Renaming it "scale", or

2) Mentioning what it represents in the documentation.

**Comment by pomerlef** [comment]
I agree with the idea. It might break few things along the way (people building there own point cloud). Maybe keep it backward compatible?
